### PR TITLE
added correct suma version in bootstrap script

### PR DESCRIPTION
**Pull request by atighineanu**
## What does this PR change?
This PR maxes the generated bootstrap script for traditional clients have the proper SUMA version echoed. 

## GUI diff

No difference.

Before:
When checking the bootstrap script for traditional clients in **/srv/www/htdocs/pub/bootstrap** 
```
#!/bin/bash
echo "SUSE Manager Server Client bootstrap script v4.1"
```

After:

It will echo the right version of SUMA Client.

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
LGTM, thanks.

But this is not a test suite change, so please use developers as reviewers.

File: spacewalk/certs-tools/spacewalk-certs-tools.changes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- added correct SUMA version in traditional client bootstr. script
```
Comment (juliogonzalez) [inline spacewalk/certs-tools/spacewalk-certs-tools.changes:1]: You missed the empty line :-)

Also write `bootstrap` script, the full name :-)

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Small change required, and ready to go.

File: spacewalk/certs-tools/spacewalk-certs-tools.changes (hunk 1 above)
Comment (atighineanu) [reply spacewalk/certs-tools/spacewalk-certs-tools.changes]: Done. Is it okay now? :)

**Review by juliogonzalez — APPROVED** [review]
Perfect. Merging.

**Pull request merged**